### PR TITLE
Simple HTTP authentication

### PR DESCRIPTION
**Pull request by sergii-tkachenko**
#### What's this PR do?
- Blink simple HTTP authentication, default user `blink`, password `blink`
- Now with 100% more Doctor Who references

#### How should this be manually tested?
- `yarn install`
- `yarn all-tests`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,11 +15,12 @@ Install dependencies `yarn install`
...
 - Launch RabbitMQ `docker-compose up`
 - `yarn web`
-- Open `http://localhost:5050`
+- Open `http://localhost:5050`  
+  Username: `blink`, password: `blink`.
 
 #### Available services
 - `localhost:5672`: RabbitMQ AMQP
-- [`localhost:15672`](http://localhost:15672): RabbitMQ management.
+- [`localhost:15672`](http://localhost:15672): RabbitMQ management.
```
Comment (aaronschachter) [inline README.md]: Trailing space?

Comment (sergii-tkachenko) [inline README.md]: No, this is actually a part of Markdown syntax. This means line break.

Comment (sergii-tkachenko) [inline README.md]: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#line-breaks

Comment (aaronschachter) [inline README.md]: Oooh. 👍

Comment (sergii-tkachenko) [inline README.md]: Actually, they recommend against it now, will change.

Comment (sergii-tkachenko) [inline README.md]: Nope, turned out, it's still recommended in lists: 

> To have a line break without a paragraph, you will need to use two trailing spaces.
> Note that this line is separate, but within the same paragraph.

**Pull request merged**